Sources/StreamChatUI/MessageActionsPopup/ChatMessageActionsVC.swift

```diff
@@ -81,9 +81,11 @@ open class ChatMessageActionsVC: _ViewController, ThemeProvider {
 
         switch message.localState {
         case nil:
-            var actions: [ChatMessageActionItem] = [
-                inlineReplyActionItem()
-            ]
+            var actions: [ChatMessageActionItem] = []
+            
+            if channelConfig.quotesEnabled {
+                actions.append(inlineReplyActionItem())
+            }
```

Comment:
Can we add a test for this in `ChatMessageActionsVC_Tests.swift`? We have an example already for "mutesEnabled", should be a simple one 👍

Reply:
Yes, good point. Added now 👍